environment.yml

```diff
@@ -41,7 +41,7 @@ dependencies:
   - ninja==1.7.2
   - conda-forge/label/gcc7::nn
   - opencv>=3.0.0,<=4.0.0
-  - openssl
+  - openssl>=1.0.0,<1.1.0
```

Comment:
why not just openssl=1.0.0?

Reply:
conda-forge actual serves version 1.0.2t

I put a range on it in hopes that it will be a bit more flexible if something changes. Since this is related to our custom builds should i just pin it to 1.0.2t?

Reply:
If we can only get a successful build with one version then yes, but if there is hardly a functional difference between them (the builds probably cares more that there exists a lib with the 1.0.0 version suffix) then keep it as a range.

Reply:
I haven't tested other versions, but the issue is mostly that xerces is looking for libssl1.0.0 which all of the different versions are named. So, the range should handle them all.